internal/rollbar/rollbar.go

```diff
@@ -77,8 +94,9 @@ func Wait() { rollbar.Wait() }
 
 func logToRollbar(critical bool, message string, args ...interface{}) {
 	// only log to rollbar when on release, beta or unstable branch and when built via CI (ie., non-local build)
-	isPublicChannel := (constants.BranchName == constants.ReleaseBranch || constants.BranchName == constants.BetaBranch || constants.BranchName == constants.ExperimentalBranch)
-	if !isPublicChannel || !condition.BuiltViaCI() {
+	isPublicChannel := constants.BranchName == constants.ReleaseBranch || constants.BranchName == constants.BetaBranch || constants.BranchName == constants.ExperimentalBranch
+	reportingDisabled := currentCfg != nil && !currentCfg.Closed() && currentCfg.IsSet(constants.ReportErrorsConfig) && !currentCfg.GetBool(constants.ReportErrorsConfig)
+	if !isPublicChannel || !condition.BuiltViaCI() || !reportingDisabled {
```

Comment:
Please evaluate this during construction (eg. when the config is provided).

Reply:
Okay.